docs/getting-started.md

```diff
@@ -118,8 +118,8 @@ Note. De prerequisite van OpenAPI Generator is JAVA. Je moet een JAVA runtime in
 
 ## Probeer en test de API
 De werking van de API is het makkelijkst te testen met behulp van [Postman](https://www.getpostman.com/).
-We hebben al een project voor je gemaakt die je kan gebruiken: [BRK-Bevragen-postman-collection.json](https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/blob/master/test/BRK-Bevragen-postman-collection.json). Hierin moet je alleen de endpoints en authenticatie (API-key) nog invullen.
-We hebben al een [Postman collection](https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/blob/master/test/BRK-Bevragen-postman-collection.json){:target="_blank" rel="noopener"} voor je klaargezet. Deze kun je importeren in Postman.
+De [openapi.yaml](https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/blob/master/specificatie/genereervariant/openapi.yaml){:target="_blank"} kun je importeren als project, waarna de verschillende requests worden ingeladen die deze API ondersteunt.
```

Comment:
Is de {target="_blank"}  echt nodig. Als je gewoon in Github kijkt zie je deze ook. En kan deze niet binnen de =haakjes ? of werkt dat niet (niet geprobeerd)

Reply:
Nee, het moet zoals hier getoond, dit is zoals je dit in Markdown doet.
Binnen de docs folder heb ik er zelf geen probleem mee dat dit er in staat. Die folder is immers bedoeld voor GitHub Pages. In de README.md laat ik dit echter steeds weg (tenminste dat is de bedoeling).

Of het echt nodig is moeten we gezamenlijk bepalen. We hebben binnen het team Standaarden echter wel besloten dat we de Redoc en Swagger interfaces in een nieuw tabblad willen openen en dan zul je deze notatie moeten gebruiken.